### ISSUE TITLE
Ombi requests not showing on Homepage

### ISSUE THREAD

**Issue by kevinb77** [causefx/Organizr#795]
INFORMATION
Organizr Version 2.0.0-alpha.250
Organizr Branch v2-master
Database Location C:/nginx/html/BerardDB.db
PHP Version 7.1.8
Webserver Version nginx/1.13.6
Windows 10

I am not able to see Ombi requests on home page or anywhere for that matter

I have tried entering several variants of the IP address for my Ombi server:
localhost
actual IP
192
noip address

Nothing seems to work.

Is this a known bug?

**Comment by causefx** [comment]
Ombi isn't setup from v2 yet

**Comment by kevinb77** [comment]
Well that answers that.

Thanks for the quick response. Any eta on setup?

**Comment by causefx** [comment]
Maybe a week or two. Need to plan out some other things.

**Comment by riemers** [comment]
I do see a request though, although it is not complete.

![image](https://user-images.githubusercontent.com/682041/37273946-8902f610-25db-11e8-8e83-ef71b72c3dbc.png)

Presume it is because of these errors i see in the homepage:
```
Warning: Illegal string offset 'posterPath' in /config/www/Dashboard/functions.php on line 5046

Warning: Illegal string offset 'theMovieDbId' in /config/www/Dashboard/functions.php on line 5048

Warning: Illegal string offset 'title' in /config/www/Dashboard/functions.php on line 5049

Warning: Illegal string offset 'posterPath' in /config/www/Dashboard/functions.php on line 5050

Warning: Illegal string offset 'posterPath' in /config/www/Dashboard/functions.php on line 5050

Warning: Illegal string offset 'approved' in /config/www/Dashboard/functions.php on line 5051

Warning: Illegal string offset 'available' in /config/www/Dashboard/functions.php on line 5052

Warning: Illegal string offset 'denied' in /config/www/Dashboard/functions.php on line 5053

Warning: Illegal string offset 'deniedReason' in /config/www/Dashboard/functions.php on line 5054

Warning: Illegal string offset 'requestedUser' in /config/www/Dashboard/functions.php on line 5055

Warning: Illegal string offset 'userName' in /config/www/Dashboard/functions.php on line 5055

Warning: Illegal string offset 'id' in /config/www/Dashboard/functions.php on line 5056

Warning: Illegal string offset 'requestedDate' in /config/www/Dashboard/functions.php on line 5057

Warning: Illegal string offset 'releaseDate' in /config/www/Dashboard/functions.php on line 5058

Warning: Illegal string offset 'childRequests' in /config/www/Dashboard/functions.php on line 5067
```

Will wait for an update, keep up the good work.

**Comment by christronyxyocum** [comment]
@riemers What you're seeing is due to an issue in Ombi itself. That has since been corrected so please update your Ombi install.

**Comment by kevinb77** [comment]
What version of Ombi should I have? I am on the latest versions on both 

Organizr - 2.0.0-alpha.250

and 

Ombi - Version | 3.0.2970

**Comment by riemers** [comment]
@christronyxyocum i checked again and its gone now indeed (auto updating docker containers, so it fixed itself)

**Comment by kevinb77** [comment]
I am still not seeing it, updated both apps.

**Comment by causefx** [comment]
Ombi was having API issues that i think they fixed.

**Comment by slimshizn** [comment]
Having this issue with Ombi not working on the hompage ( Org V1) Ombi devs directs me to Organizr, Organizr support directs me to Ombi. so I'm not sure who to have help me troubleshoot. I've restarted both Ombi and Organizr, checked the /reverse config, everythings fine. Ombi works as a tab inside Organizr just fine. I'm assuming its the api issue. I'm on the latest release of Ombi.

**Comment by causefx** [comment]
Can you define what isn’t working? Screenshot plz

**Comment by slimshizn** [comment]
![ombi](https://user-images.githubusercontent.com/23506499/39655134-1cce10c8-4fc6-11e8-8baf-0f9befea977b.png)
I've tried adding a trailing slash at the end of both homepage and tab settings, that doesn't allow sso to work. I've tried just tab, that changes nothing at all. Just homepage settings doesn't allow sso. The nginx Ombi conf I have here was given to me by someone that has Ombi working on the homepage.
